knife

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env bash
```

Comment:
I think I would prefer these be separate scripts that had more descriptive names rather than knife?

Reply:
naming is tbd, i can split them as well.